services/core-api/app/api/verify/namespace.py

```diff
@@ -0,0 +1,9 @@
+from flask_restplus import Namespace
+
+from app.api.verify.permit.now.resources.verify_permit_now_resource import VerifyPermitNOWResource
+from app.api.verify.permit.mine.resources.verify_permit_mine_resource import VerifyPermitMineResource
+
+api = Namespace('verify', description='Verification endpoints')
+
+api.add_resource(VerifyPermitNOWResource, '/permit/now')
```

Comment:
Should this be `/permit/now/verification`. Or rather, I'd not want to have a `now` endpoint that doesn't return our `now` object.

Reply:
I think this should go under an explicit /integration path because it doesn't follow our API conventions, but @Jsyro and I compromised on /verify/permit as following our resource path conventions while also being seperate.

Reply:
Sorry, didn't see the namespace, makes sense.